### ISSUE TITLE
Optional value comment

### ISSUE THREAD

**Issue by damienmckenna** [pantheon-systems/terminus-secrets-manager-plugin#31]
It would be *very* useful if an item could be added as a comment, e.g.

    terminus secret:site:set <site> somevariable mango --comment="Overrides a value normally stored in configuration; see WTF-2345 for details."

That would end up as something like the following in the JSON file:

    {
      "key": "somevariable",
      "value": "mango",
      "comment": "Overrides a value normally stored in configuration; see WTF-2345 for details."
    }

**Comment by greg-1-anderson** [comment]
Good idea. Tracking internally as https://getpantheon.atlassian.net/browse/FEAT-1145
